autogalaxy/operate/image.py

```diff
@@ -372,12 +376,12 @@ class OperateImageGalaxies(OperateImageList):
 
     def galaxy_image_2d_dict_from(
         self, grid: aa.Grid2D, operated_only: Optional[bool] = None

```

Comment:
Why is operated_only optional rather than defaulting to False?

If you want three possible values it is more explicit to use an [enumerator](https://docs.python.org/3/library/enum.html)